### PR TITLE
[ Merge after PR #1 ] Add Hack CU IV date & location!!

### PR DESCRIPTION
**Pull request by elliotwhitehead**
## Description
Adds date and location to HackCU IV event footer! 🎉 

image

## Some questions
- [x] I read the contributing guidelines
- [ ] I'm adding a new event/footer link
- [x] I'm editing an existing event/footer link
- [ ] I'm adding a new feature/fixing a bug

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -38,7 +38,8 @@
   url: https://local.hackcu.org
   
 - name: HackCU IV
-  location: TBD
+  date: 2018-02-24 8:00 AM
+  location: SEEC Building (East Campus)
```
Comment (AparaV) [inline _data/events.yml]: Can we add a hyperlink to a map or something? That would make it easier for people who are not familiar with the building (like me!) to find it.

File: _data/events.yml [hunk 2]
```diff
@@ -38,7 +38,8 @@
...
   image-url: assets/img/hackcu.png
   line: Subscribe to be the first to know when applications go live.
```
Comment (AparaV) [inline _data/events.yml]: I would suggest that we wait until @casassg deploys the backend for public. And then update the description also in the same PR

**Review by AparaV — CHANGES_REQUESTED** [review]
Let's wait until we deploy the registration backend. Then we can keep the entire update of the event (along with the description) to the same PR.

Or, you can update the description now (not waiting on the applications to go live) and merge it.

File: _data/events.yml (hunk 1 above)
Comment (AparaV) [reply _data/events.yml]: Never mind, just realized that we have a link to the actual website where we will soon have the map of the venue

File: _data/events.yml [hunk 3]
```diff
@@ -38,7 +38,8 @@
   url: https://local.hackcu.org
   
 - name: HackCU IV
-  location: TBD
+  date: 2018-02-24 8:00 AM
+  location: SEEC Building (East Campus)
   image-url: assets/img/hackcu.png
```
Comment (casassg) [inline _data/events.yml]: Update image to something like this: 

https://beta.hackcu.org/assets/images/ogimage870x450.png

**Review by casassg — CHANGES_REQUESTED** [review]
looks good, let's also update the image preview

**Review by casassg — APPROVED** [review]

**Review by AparaV — APPROVED** [review]

**Pull request merged**